rotkehlchen/data/importer.py

```diff
@@ -182,3 +186,183 @@ def import_cointracking_csv(self, filepath: Path) -> None:
                     continue
 
         return None
+
+    def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
+        """Consumes a cryptocom entry row from the CSV and adds it into the database
+        Can raise:
+            - DeserializationError if something is wrong with the format of the expected values
+            - UnsupportedCryptocomEntry if importing of this entry is not supported.
+            - IndexError if the CSV file is corrupt
+            - UnknownAsset if one of the assets founds in the entry are not supported
+        """
+        row_type = csv_row['Transaction Kind']
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['Timestamp (UTC)'],
+            formatstr='%Y-%m-%d %H:%M:%S',
+            location='crypto.com',
+        )
+        description = csv_row['Transaction Description']
+        notes = f'{description}\nSource: crypto.com (CSV import)'
+
+        # No fees info for now (Aug 2020) on crypto.com, so we put 0 fees
+        fee = Fee(ZERO)
+        fee_currency = A_USD  # whatever (used only if there is no fee)
+
+        if row_type in (
+            'crypto_purchase',
+            'crypto_exchange',
+            'referral_gift',
+            'crypto_earn_interest_paid',
+        ):
+            # variable mapping to raw data
+            currency = csv_row['Currency']
+            to_currency = csv_row['To Currency']
+            native_currency = csv_row['Native Currency']
+            amount = csv_row['Amount']
+            to_amount = csv_row['To Amount']
+            native_amount = csv_row['Native Amount']
+
+            trade_type = TradeType.BUY if to_currency != native_currency else TradeType.SELL
+
+            if row_type in ('crypto_exchange'):
```

Comment:
If you are comparing only one string to the other as you are doing here just use equality comparison. So

```
if row_type == 'crypto_exchange':
    ...
```